README.md

```diff
@@ -1,164 +1,134 @@
 
-Verifiers can check that these signatures are valid and associated with the signing OpenID identity. OpenPubkey does not add any new trusted parties beyond what is required for OpenID Connect. It is fully compatible with existing OpenID Providers (Google, Azure/Microsoft, Okta, OneLogin, Keycloak) without any changes to the OpenID Provider.
+OpenPubkey does not add any new trusted parties beyond what is required for OpenID Connect. It is fully compatible with existing OpenID Providers (Google, Azure/Microsoft, Okta, OneLogin, Keycloak) without any changes to the OpenID Provider.
 
-### ID Tokens and PK Tokens
+OpenPubkey can be used to enable a myriad of applications. For example, companies building on OpenPubkey include:
 
-OpenPubkey uses ID Tokens issued by OpenID Providers (OPs) to produce PK Tokens. A PK Token consist of the ID Token and the CIC (Client-Instance Claims) where the CIC contains the user's public key and associated metadata. The ID Token contains a hash of the CIC, so that the values in the CIC including the identity's public key are cryptographically bound to the ID Token signed by the OP.
+* [Docker, Inc](https://www.docker.com/) is building a public container registry [with OpenPubkey used to sign Docker Official Images](https://www.docker.com/blog/signing-docker-official-images-using-openpubkey/).
 
-For workload-identity, the hash of the CIC is stored in the ID Token's `aud` claim following the pattern of GitHub Action's workload identity. For user-identity, the hash of the CIC is stored in the ID Token's `nonce` claim.
+* [BastionZero](https://www.bastionzero.com/) uses OpenPubkey to provide secure remote access to infrastructure.
 
-## OpenPubkey FAQ
+OpenPubkey is a Linux Foundation project open-sourced under the Apache 2.0 license. This project presently provides an OpenPubkey client and verifier for creating and verifying PK Tokens from Google’s OP (for users) and GitHub’s OP (for workloads).
 
-### What is the difference between Sigstore and OpenPubkey?
+## How Does OpenPubkey Work?
 
-OpenPubkey cannot really be compared to Sigstore. Sigstore is an end-to-end artifact signing solution, whereas OpenPubkey only binds public keys to OIDC identities for use as part of a larger signing solution. OpenPubkey is complementary to Sigstore, for example https://github.com/sigstore/fulcio/issues/1056
+### OpenPubkey and User Identities
 
-As stated in the Related Work section of [the OpenPubkey paper](https://eprint.iacr.org/2023/296.pdf):
+OpenPubkey supports both workload identities and user identities. Let's look at how this works for users and then show how to extend OpenPubkey to workloads.
```

Comment:
If we are going to include the subheading `### OpenPubkey and User Identities` this I propose we cut my two sentences here. I only kept them because I cut the subheading.

```suggestion

```

Reply:
Fixed. Meant for this sentence to be directly under the major header. Thanks for the catch, Ethan